csrc/mmdeploy/codebase/mmcls/multi_label_linear_cls.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#include <algorithm>
+#include <numeric>
+
+#include "mmdeploy/codebase/mmcls/mmcls.h"
+#include "mmdeploy/core/tensor.h"
+#include "mmdeploy/core/utils/device_utils.h"
+#include "mmdeploy/core/utils/formatter.h"
+#include "mmdeploy/experimental/module_adapter.h"
+
+using std::vector;
+
+namespace mmdeploy::mmcls {
+
+class MultiLabelLinearClsHead : public MMClassification {
+ public:
+  explicit MultiLabelLinearClsHead(const Value& cfg) : MMClassification(cfg) {
+    if (cfg.contains("params")) {
+      num_classes_ = cfg["params"].value("num_classes", 1);
```

Comment:
`num_classes` is useless in this context. Remove the parsing of `num_classes` and member `num_classes_`. Use `class_num` which is computed from the output shape of the model instead.